wavefront/server/modules/knowledge_base_module/knowledge_base_module/controllers/rag_retreival_controller.py

```
@@ -96,6 +99,112 @@ def convert_uuids_to_str(data):
+        return (
+            None,
+            JSONResponse(
+                status_code=status.HTTP_400_BAD_REQUEST,
+                content=response_formatter.buildErrorResponse(
+                    'image_url s3:// is only supported when cloud provider is AWS'
+                ),
+            ),
+        )
+    try:
+        content = cloud_storage.read_file(bucket, key)
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)